veg-index/utils.py

```
@@ -0,0 +1,233 @@
+from __future__ import annotations
+
+import os
+from contextlib import asynccontextmanager, suppress
```

Ruff (0.11.9)

`contextlib.asynccontextmanager` imported but unused

Remove unused import: `contextlib.asynccontextmanager`

(F401)